integration/configs.go

```diff
@@ -51,9 +51,20 @@ var (
 	cortexSchemaConfigYaml = buildSchemaConfigWith([]storeConfig{{From: "2019-03-20", IndexStore: "aws-dynamo"}})
 
 	AlertmanagerFlags = map[string]string{
-		"-alertmanager.storage.local.path": filepath.Join(e2e.ContainerSharedDir, "alertmanager_configs"),
+		"-alertmanager.configs.poll-interval": "1s",
+		"-alertmanager.web.external-url":      "http://localhost/api/prom",
```

Comment:
```suggestion
		"-alertmanager.web.external-url":      "http://localhost/alertmanager",
```

It might be worth updating this to use the URL that is active in both the `alertmanager` target and the `all` target